utils/pkg/pkg_run

```diff
@@ -158,9 +162,22 @@ package_cloud_push () {
       export PC_REPO="worker-testing"
     fi
 
-    package_cloud push \
-      $PC_USER/$PC_REPO/$package_cloud_os/$platform_release \
-      tmp/output/$pkgtype/$platform_family/$platform_release/*.$pkgtype --verbose
+    PC_UPLOAD_FILE=$(ls $CHECKOUT_ROOT/tmp/output/$pkgtype/$platform_family/$platform_release/*.$pkgtype)
```

Comment:
(if not, I'd just `echo whatever/*.glob` directly instead and let the shell do its thing)